page/sass/product.scss

```diff
@@ -0,0 +1,255 @@
+[data-entrance] {visibility: hidden;} // required for JS animation to work
```

Comment:
could you wrap everything in .product-page class? (in html as well) so id doesnt leak to other sites